draft-ietf-capport-api.md

```diff
@@ -122,7 +123,8 @@ Content-Type: application/captive+json
    "captive": true,
    "user-portal-url": "https://example.org/portal.html",
    "venue-info-url": "https://flight.example.com/entertainment",
-   "expire-date": "2014-01-01T23:28:56.782Z"
+   "expire-date": "2014-01-01T23:28:56.782Z",
```

Comment:
should this be changed to seconds-remaining?

Reply:
Good catch, fixed!